kitsune/products/migrations/0015_auto_20240722_0449.py

```diff
@@ -0,0 +1,527 @@
+# Generated by Django 4.2.14 on 2024-07-22 04:49
+
+from django.db import migrations
+
+TAXONOMY_MAPPING = [
+    {
+        "old_title": "Fix problems with websites",
+        "old_slug": "websites",
+        "title": "Site breakages",
+        "slug": "site-breakages",
+        "parent_topic_slug": "performance-and-connectivity",
+    },
+    {
+        "old_title": "Firefox crashes",
+        "old_slug": "crashing",
+        "title": "Crashing and slow performance",
+        "slug": "crashing-and-slow-performance",
+        "parent_topic_slug": "performance-and-connectivity",
+    },
+    {
+        "old_title": "Unblock Firefox from connecting to the internet",
+        "old_slug": "unblock-firefox-connecting-internet",
+        "title": "Performance and connectivity",
+        "slug": "performance-and-connectivity",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Firefox won't save settings or remember information",
+        "old_slug": "firefox-wont-save-settings-or-remember-information",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Problems with add-ons, plugins or unwanted software",
+        "old_slug": "problems-add-ons-plugins-or-unwanted-software",
+        "title": "Add-ons, extensions, and themes",
+        "slug": "add-ons-extensions-and-themes",
+        "parent_topic_slug": "settings",
+    },
+    {
+        "old_title": "Videos, sound, pictures and animations don't work",
+        "old_slug": "videos-sound-pictures-and-animations-dont-work",
+        "title": "Multimedia",
+        "slug": "multimedia",
+        "parent_topic_slug": "browse",
+    },
+    {
+        "old_title": "Basic browsing",
+        "old_slug": "basic-browsing-firefox",
```

Comment:
This entry is great, but I noticed two very similar old topics that aren't covered:
- `basic-browsing-firefox-android`
- ``basic-browsing-firefox-ios`

Should we create similar entries for those as well?